lexicons/tools/ozone/moderation/defs.json

```diff
@@ -182,6 +182,76 @@
         "tags": {
           "type": "array",
           "items": { "type": "string" }
+        },
+        "accountStats": {
+          "description": "Statistics related to the account subject",
+          "type": "ref",
+          "ref": "#accountStats"
+        },
+        "recordsStats": {
+          "description": "Statistics related to the record subjects authored by the subject's account",
+          "type": "ref",
+          "ref": "#recordsStats"
+        }
+      }
+    },
+    "accountStats": {
+      "description": "Statistics about a particular account subject on the labeller",
```

Comment:
nit, but generally we refer to these as "mod service" instead of labeler since it's duties are wider than just labeling. alternately, could just leave off the "on the labeler"